submissions/morenko/memory-game/js/index.js

```diff
@@ -0,0 +1,182 @@
+// создадим массив с картами
+const listCards = [
+    {
+        name: 1,
+        front: 'assets/front/1.jpg'
+    },
+    {
+        name: 2,
+        front: 'assets/front/2.jpg'
+    },
+    {
+        name: 3,
+        front: 'assets/front/3.jpg'
+    },
+    {
+        name: 4,
+        front: 'assets/front/4.jpg'
+    },
+    {
+        name: 5,
+        front: 'assets/front/5.jpg'
+    },
+    {
+        name: 6,
+        front: 'assets/front/6.jpg'
+    },
+    {
+        name: 7,
+        front: 'assets/front/7.jpg'
+    },
+    {
+        name: 8,
+        front: 'assets/front/8.jpg'
+    },
+];
+//console.log(cards);
+
+// создадим копию массива
+const copylistCards = listCards.slice();
+//console.log(copyCards);
+
+const cards = listCards.concat(copylistCards); //объединим два одинаковых массива карт
+//console.log(cards); //проверим
+let countCards = cards.length; // длина массива - количество карт в нем
```

Comment:
Why are you saving this into a variable? Length can be changed and you need initial value?